components/ai_chat/renderer/page_text_distilling.cc

```diff
@@ -139,8 +144,29 @@ void AddTextNodesToVector(const ui::AXNode* node,
 
 void DistillPageText(
     content::RenderFrame* render_frame,
+    int32_t global_world_id,
     int32_t isolated_world_id,
     base::OnceCallback<void(const std::optional<std::string>&)> callback) {
+  blink::WebLocalFrame* main_frame = render_frame->GetWebFrame();
+
+  if (ai_chat::features::IsAIHostSpecificDistillationEnabled()) {
```

Comment:
nit: I think this would be better if we just did at the beginning of the function something like:

```cxx
if (ai_chat::features::IsAIHostSpecificDistillationEnabled()) {
  return;
}
```

This way we wouldn't be wrapping the whole function inside this block.

Reply:
There are other things that happen here if it's disabled, but now that I'm looking at it again I think it could probably be re-organized to be more clear that it either does `DistillPageTextViaSiteScript` or `CreateAXTreeSnapshotter` depending on both the flag and whether or not their is a site script

Reply:
Ah, I missed that.

Reply:
Pushed a change to reconcile the naming patterns. Thanks!